cpp/src/arrow/io/buffered_test.cc

```diff
@@ -293,6 +293,14 @@ TEST_F(TestBufferedOutputStream, Tell) {
 
   ASSERT_OK(buffered_->Close());
 
+  // write large buffer after raw_pos is cached
+  OpenBuffered(3, false);
```

Comment:
`OpenBuffered(/*buffer_size=*/3);` should make it easier to catch the intent of this test.

Reply:
done